fairlearn/reductions/_exponentiated_gradient/exponentiated_gradient.py

```diff
@@ -195,30 +200,22 @@ def _pmf_predict(self, X):
     def _format_results(self, gaps, Qs, lagrangian, B, eta_min):
```

Comment:
Can we give this a better name than _format_ results? 'Populate' might be a better verb

Reply:
Definitely! From staring at this for a while I feel like it may not be needed at all since we were just formatting results for putting them into a result object which we're not doing anymore. Stay tuned...